### ISSUE TITLE
Test explorer doesn't show tests until 'Start all runners' is executed when autoRun is set to off

### ISSUE THREAD

**Issue by obambrough** [jest-community/vscode-jest#911]
### Environment

1. `vscode-jest version`: 4.6.0
2. `node -v`: 16.16.0
3. `npm -v` or `yarn --version`: yarn v1.22.18
4. `npm ls jest` or `npm ls react-scripts` (if you haven’t ejected): [fill]
5. your vscode-jest settings if customized:
    - jest.jestCommandLine: "node_modules/.bin/jest",
    - jest.autoRun: "off",
    - jest.rootPath: "components/ui",
6. Operating system: MacOS 12.5.1

### Prerequisite
- are you able to run jest test from the command line? Yes
- how do you run your tests from the command line? `yarn test`

### Steps to Reproduce

1. In project's workspace settings configure jest.autoRun to be 'off'
2. Reopen the project in VS Code
3. Open the 'Test Explorer' side panel, the Jest unit tests are not shown
4. Run the 'Jest: Start All Runners' command
5. The Text Explorer is now populated with the Jest unit tests (but, as expected, they haven't been run.)

### Expected Behavior

The Jest unit tests should be shown in the Test Explorer without having to run the 'Jest: Start All Runners' command. Microsoft's Python extension populates the Test Explorer with unit tests without needing any additional command to be run. 

### Actual Behavior

The Jest unit tests are missing from the Test Explorer when 'jest.autoRun' is set to 'off'.

**Comment by connectdotz** [comment]
hi @obambrough a few questions to help us investigate:

1. in step 3, did you see the test files listed similar to the image attached below? did you see a "Jest: ..." in your status bar below? This will tell us if the extension is activated.
<img width="360" alt="Screen Shot 2022-10-02 at 4 04 44 PM" src="https://user-images.githubusercontent.com/891093/193473974-eedd49da-cf8a-48aa-86c4-339dda831e92.png">

2. in step 5, did you see all the individual tests or just the test files like in the attached image above?

**Comment by arcataroger** [comment]
I'm running into a similar situation here, _even with autoRun enabled_:
```  "jest.autoRun": { "watch": true, "onStartup": ["all-files"], "onSave": "test-src-file" },
  "jest.jestCommandLine": "yarn test",
  "jest.rootPath": "packages/app-v1",
  "jest.testExplorer": {
    "showInlineError": true
  }```

On startup, the Test Explorer doesn't show up at all:
<img width="52" alt="image" src="https://user-images.githubusercontent.com/11964962/194641939-629c6ae0-e2dc-433a-989f-6e09685c718c.png">


Or in the status bar:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/11964962/194641962-beb21250-a363-4cb5-ac60-12e93dcc2ff7.png">


Only when I manually invoke the "Jest: Start all runners" command does the Test Runner and status bar appear, and then everything works fine after that.

**Comment by connectdotz** [comment]
hi, yes these seem to be the activation issues, which should be fixed in the [v5.0.0 pre-release](https://github.com/jest-community/vscode-jest/releases/tag/v5.0.0). We just released it today, please feel free to give it a try and let us know how it works for you. thanks.

**Comment by obambrough** [comment]
Awesome, I tested v5.0.0 and it did successfully activate on startup - thanks!
